feature/login/src/main/java/com/epmedu/animeal/login/code/presentation/EnterCodeScreen.kt

```diff
@@ -36,8 +35,8 @@ internal fun EnterCodeScreen() {
     LaunchedEffect(Unit) {
         focusRequester.requestFocus()
 
-        viewModel.isCodeCorrect.collectLatest { isCorrect ->
-            if (isCorrect) {
+        viewModel.event.collect {
+            if (it is EnterCodeViewModel.Event.NavigateToFinishProfile) {
```

Comment:
Could you please make import for `EnterCodeViewModel` 

will be
```kotlin
if (it is Event.NavigateToFinishProfile) {
```

Reply:
Updated